### PR TITLE
Sa 60/cypress tests/init

### PR DESCRIPTION
**Pull request by sairaghav**
# Description

Added initial tests for homepage, upload modal, about page and error page.
Added cypress package to the dependencies
https://smarticle.atlassian.net/wiki/spaces/SA/pages/13205506/Testing+Documentation

## Type of change

Please delete options that are not relevant.
- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## Screenshots

image

# Reference JIRA tickets : 

- SA-60

# Checklist:

Put an "x" in the brackets to check the checkbox.

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code in particularly hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] I have made corresponding changes to JIRA tickets
- [ x] My changes generate no new warnings
- [ x] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes
- [ x] I have merged develop into my branch and resolved possible conflicts

### CODE REVIEW

File: cypress.config.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  e2e: {
+    baseUrl:
+      // process.env.REACT_APP_ENV === "prod"
```
Comment (DarioN555) [inline cypress.config.ts]: Why is this commented out? Should we add the `REACT_APP_ENV` variable to .env set as `prod` and add another `.env.local` file for local use which has it set to `local`.  The `.env.local` file takes precedence over the `.env` file, but only exists locally and should be ignored by git.

**Review by DarioN555 — APPROVED** [review]
Great job, really like how this is setup!

File: cypress.config.ts (hunk 1 above)
Comment (sairaghav) [reply cypress.config.ts]: That is not necessary. I was trying the tests out for prod website manually and forgot to remove the comment

**Review by nMedic246 — APPROVED** [review]
Good job, it works well!

**Pull request merged**